src/main/frontend/components/VenueTile.js

```diff
@@ -1,22 +1,18 @@
 import React from "react";
 
 const VenueTile = (props) => {
-  const {
-    name,
-    city,
-    state,
-    address,
-    zip,
-    phoneNumber,
-    capacity,
-    description,
-    imgUrl,
-  } = props.venue;
+  const { id, name, description, imgUrl } = props.venue;
 
   return (
     <div>
-      <h2>{name}</h2>
-      <img src={imgUrl} />
+      <a href={`/concert-venues/${id}`}>
+        <h2>{name}</h2>
+      </a>
+
+      <a href={`/concert-venues/${id}`}>
+        <img src={imgUrl} />
+      </a>
```

Comment:
We can use `Link` components instead of anchor tags to take full advantage of React

Reply:
These still need to be updated to `<Link/>` components